configure.ac

```diff
@@ -923,6 +923,20 @@ AS_IF([test "x$enable_slingshot" = xyes],[
         [AC_MSG_ERROR([libcxi or its headers not found])])
 ])
 
+
+AC_ARG_ENABLE([libzfs],
+    [AS_HELP_STRING([--enable-libzfs], [require the libzfs related plugins @<:@default=check@:>@])],
+    [],
+    [enable_libzfs="check"])
+AM_CONDITIONAL([ENABLE_LIBZFS], [test "x$enable_libzfs" != xno])
+AM_CONDITIONAL([HAVE_LIBZFS], [test "x$have_libzfs" != xno])
```

Comment:
This is in the wrong place, because you haven't done anythin that would set "have_libzfs" yet. It will always be true. But lets just remove this AM_CONDITIONAL altogether.